spec/schema.json

```diff
@@ -410,6 +416,23 @@
         "SUBMITTED",
         "PROCESSED"
       ]
+    },
+    "Ineligible": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "$ref": "#/definitions/IneligibilityStatus"
+        }
+      }
+    },
+    "IneligibilityStatus": {
+      "type": "string",
+      "enum": [
+        "INELIGIBLE_COMPANY_STRUCK_OFF",
+        "INELIGIBLE_COMPANY_SECOND_GAZETTE",
+        "INELIGIBLE_COMPANY_CONVERTED",
```

Comment:
i think INELIGIBLE_COMPANY_CONVERTED  should be INELIGIBLE_COMPANY_CONVERTED_OR_CLOSED

Reply:
Yeh, I didn't want to make them too long so tried to abbreviate as best I could, but can change if we are happy with having the 4 codes - see Martin's comment above.